docs/swagger/spec.yml

```diff
@@ -466,32 +472,70 @@ paths:
         - $ref: '#/parameters/minSunAzimuth'
         - $ref: '#/parameters/maxSunElevation'
         - $ref: '#/parameters/minSunElevation'
-        - $ref: '#/parameters/tags'          
+        - $ref: '#/parameters/tags'
       responses:
         200:
           description: Paginated list of scenes associated with this bucket
           schema:
             $ref: '#/definitions/ScenePaginated'
     post:
-      summary: Create a new association between a scene and this bucket
+      summary: Create a new association between a set of scenes and this bucket
       tags:
         - Imagery
       parameters:
         - $ref: '#/parameters/uuid'
-        - name: image
+        - name: scenes
           in: body
           required: true
+          description: UUIDs of scenes to replace in bucket
```

Comment:
"place" not "replace". this action should return the same result if none of the listed scenes are in the bucket as when all of them are.